metadata-io/src/main/java/com/linkedin/metadata/entity/AspectUtils.java

```diff
@@ -72,20 +72,35 @@ public static Map<Urn, Aspect> batchGetLatestAspect(
 
   private static MetadataChangeProposal getProposalFromAspect(String aspectName, RecordTemplate aspect,
       MetadataChangeProposal original) {
-    try {
-      MetadataChangeProposal proposal = original.copy();
-      GenericAspect genericAspect = GenericRecordUtils.serializeAspect(aspect);
-      // Additional changes should never be set as PATCH, if a PATCH is coming across it should be an UPSERT
-      if (ChangeType.PATCH.equals(proposal.getChangeType())) {
-        proposal.setChangeType(ChangeType.UPSERT);
-      }
-      proposal.setAspect(genericAspect);
-      proposal.setAspectName(aspectName);
-      return proposal;
-    } catch (CloneNotSupportedException e) {
-      log.error("Issue while generating additional proposals corresponding to the input proposal", e);
+    MetadataChangeProposal proposal = new MetadataChangeProposal();
```

Comment:
Why were these changes needed for this PR?

Reply:
It's a long time bug that surfaces due to the changes made here. Basically what happens here is that the underlying datamap gets reused from the change proposal and certain fields get passed through that cause issues. This change removes that bug and allows the patches to go through whenever it's the first modification to the aspect.